### ISSUE TITLE
Select "Development Request" for Active Workflow Development

### ISSUE THREAD

**Issue by Rhynorater** [caido/caido#1308]
### Community Note

- Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help us know what to work on
- Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request

### Description

When developing an Active Workflow, we constantly have to click out of the Active Workflow tab and select a request, right click, "Run Workflow", select our workflow, then check the logs. We must make this lower friction for the users by selecting a specific request from replay and loading it into the Active Workflow environment to allow for easy execution of the workflow against that development request.

**Comment by Sytten** [comment]
Agreed, this will be part of the workflow refactor detailed in https://github.com/caido/caido/issues/1319
